src/Client/pages/VerbImperatives.fs

```diff
@@ -20,16 +20,16 @@ type Msg =
     | Task of Task.Msg
     
 let patterns =
-    dict [ (1, ["nést"; "číst"; "péct"; "třít"; "brát"; "mazat"])
-           (2, ["tisknout"; "minout"; "začít"])
-           (3, ["krýt"; "kupovat"])
-           (4, ["prosit"; "čistit"; "trpět"; "sázet"])
-           (5, ["dělat"]) ]
+    dict [ (E, ["nést"; "číst"; "péct"; "třít"; "brát"; "mazat"])
+           (NE, ["tisknout"; "minout"; "začít"])
```

Comment:
One question: how do we distinguish between `začít` and `třít` ? (same question regarding irregular verbs, which have shape of one class but conjugate as a different one)

Reply:
Well, short and sad answer - we don't. We detect the class not by the infinitive but by the third person singular (_začne_, _tře_)- the names of the classes refer to those endings. 

So just by infinitive one cannot for sure know the imperative of the word - they need to know the third person singular first, so they need to know some of the conjugation anyway. But this is ~~life~~Czech.

I should finally write rules in this exercise and explain things a bit.